test/fmf/tests/utils.sh

```diff
@@ -103,10 +103,10 @@ cleanup_services_configuration() {
   for service in "${services[@]}"; do
     local sysconfig_file="/etc/sysconfig/go-fdo-server-${service}"
     if [[ -f "${sysconfig_file}" ]]; then
-      # Reset ADDITIONAL_OPTS back to empty instead of deleting the file
-      if grep -q '^ADDITIONAL_OPTS=' "${sysconfig_file}"; then
-        sudo sed -i 's|^ADDITIONAL_OPTS=".*"|ADDITIONAL_OPTS=""|' "${sysconfig_file}" || true
+      # Reset CMDLINE_OPTS back to empty instead of deleting the file
+      if grep -q '^CMDLINE_OPTS=' "${sysconfig_file}"; then
+        sudo sed -i 's|^CMDLINE_OPTS=".*"|CMDLINE_OPTS=""|' "${sysconfig_file}" || true
```

Comment:
just to make sure, sysconfigs have to go away :smile: we don't want them, systemd files shouldn't reference them, everything has to come from our new shiny config yaml/toml - it's not a breaking change either so we can do whatever we want w/o breaking nobody